stylesheets/cart.css

```diff
@@ -0,0 +1,126 @@
+.cart-main {
+  margin-top: 7.5rem;
+  scroll-margin-top: 7.5rem;
+  padding: 0 5rem;
+  display: flex;
+  gap: 6rem;
+  justify-content: center;
+}
+
+.cart-active {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  align-items: center;
+}
+
+.card-horizontal {
+  max-width: none;
+  width: 40rem;
+}
+
+.card-horizontal .card-media {
+  width: 27.5%;
+}
+
+.card-price {
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+}
+
+.card-price-original {
+  text-decoration: line-through;
+}
+
+.card-discount {
+  color: var(--teal);
```

Comment:
what does teal signify? make it teal-color simplified

Reply:
Good Suggestion! Will add this in a misc Pull Request later for sure.